src/infrastructure/api/fetch_Items.ts

```diff
@@ -1,12 +1,12 @@
 import axios from 'axios';
 
-const apiCallFunc = async() => {
+const fetchItems = async() => {
   const token: string = '1k5sbhafeq3stb477djbommo9z2vrwj40x1zymsa'
   const apiCall = await axios.get('https://api.json-generator.com/templates/NM0ZuloCu1TZ/data', {headers: {"Authorization" : `Bearer ${token}`}})
```

Comment:
This fetches everything. Please change this. You need to change on json-generator so it only returns 10 items at a time. Do not fetch all 30+ items and then only show 10 at a time. That defeats the whole purpose of this excersize.

Fetch 10 at a time, and then when you click load more, it fetches 10 again and adds it to the state